include/exiv2/config.h

```diff
@@ -1,43 +1,12 @@
 // config.h
 
-#pragma once
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
-
-#ifdef  __cplusplus
-# ifndef CPLUSPLUS11
-#  define CPLUSPLUS11 201103L
-# endif
-# if     __cplusplus >= CPLUSPLUS11
-#  ifndef  EXV_USING_CPP_ELEVEN
-#   define EXV_USING_CPP_ELEVEN
-#  endif
-# endif
-#endif
-
 ///// Start of Visual Studio Support /////
 #ifdef  _MSC_VER
 
-#define _MSC_VER_2015 1900
-#define _MSC_VER_2013 1800
-#define _MSC_VER_2012 1700
-#define _MSC_VER_2010 1600
-#define _MSC_VER_2008 1500
-#define _MSC_VER_2005 1400
```

Comment:
I removed the ones for 2005 and 2003 accidentally. I have to place back those lines.